manifests/server/passenger.pp

```diff
@@ -42,11 +44,21 @@
     'path'              => "${app_root}/public/",
     'passenger_enabled' => 'On',
   }
-  
+
   $directories = [
     $directory,
   ]
 
+  $http_pre_start = $passenger_pre_start ? {
+    true  => "http://${::fqdn}:${http_port}",
```

Comment:
I guess you tested this and it's not needed, I only did read the passenger docs on that topic, but is there a / needed at the end?